docs/Installation.rst

```diff
@@ -1256,40 +1252,6 @@ to the ``CATALINA_OPTS`` variable, for example:
 |                                               | and ssh port 2000.         |
 +-----------------------------------------------+----------------------------+
 
-
-.. _eXoProfiles:
-
-================
-Startup profiles
-================
-
-eXo Platform comes with different runtime profiles, enabling you to customize
-which modules you want to enable/disable in each eXo Platform instance.
-
-.. warning:: Before modifying and developing eXo Platform, you should choose carefully
-             the profiles that are suitable to your requirements. In particular,
-			 after you have done any modifications/developments on the server
-			 that you started up with your selected profiles, and then switched
-			 to another new profiles, you will not see such
-			 modifications/developments on eXo Platform.
-
-Some eXo Platform 3.5 profiles are no longer available in eXo Platform 4
-and higer, including: *default, collaboration, social, knowledge, webos, 
-workflow*.
-Currently, eXo Platform only supports the following profiles:
-
--  ``all``: Activate all modules (such as Forum, Wiki, Calendar,
-   Social). This profile is enabled by default.
-
--  ``minimal``: Activate the Content only.
-
-To activate the ``minimal`` profile in **Tomcat**, this customized
-variable is required:
-
-::
-
-    EXO_PROFILES="minimal"
-
```

Comment:
There is a reference to this chapter in https://docs.exoplatform.org/en/5.3/Installation.html#basic-customization.

Reply:
deleted